hezar/models/text_classification/text_classification.py

```diff
@@ -0,0 +1,42 @@
+"""
+A BERT model for text classification built using HuggingFace Transformers
```

Comment:
@pooya-mohammadi Please remove this docstring

Reply:
done